build.sbt

```diff
@@ -97,9 +112,16 @@ lazy val `data-scalaxy-reader` = (project in file("."))
     publishLocal / skip := true
   )
   .aggregate(`reader-text`)
+  .aggregate(`reader-excel`)
```

Comment:
We can merge into single aggregate statement using **.aggregate(`reader-text`, `reader-excel`)**